iothub/device/tests/Amqp/AmqpTransportHandlerTests.cs

```diff
@@ -95,34 +95,6 @@ public async Task AmqpTransport_Select_CorrectReceiverLink_ForModuleTwin()
             enableReceiveMessageAsyncWasCalled.Should().BeTrue();
         }
 
-        [TestMethod]
-        [Obsolete]
-        public void AmqpTransportHandler_RejectAmqpSettingsChange()
-        {
-            // Added [Obsolete] attribute to this method to suppress CS0618 message for ConnectionIdleTimeout
-            var amqpTransportHandler1 = new AmqpTransportHandler(new PipelineContext(), new IotHubConnectionString(IotHubConnectionStringBuilder.Create(TestConnectionString)), new AmqpTransportSettings(TransportType.Amqp_Tcp_Only, 60, new AmqpConnectionPoolSettings()
-            {
-                Pooling = true,
-                MaxPoolSize = 10,
-                ConnectionIdleTimeout = TimeSpan.FromMinutes(1)
```

Comment:
For some context, this test checks that amqp settings changes are rejected, which is still a valid scenario. The obsolote part of this test is the fact that ConnectionIdleTimeout is being set as one of the setting's properties. You could simply remove ConnectionIdleTimeout and remove the obsolete tag.